### ISSUE TITLE
notice: use "wd deploy --check" instead of "check='yes' wd deploy" since gem version 0.5.2

### ISSUE THREAD

**Issue by rick** [flogic/whiskey_disk#3]
tl;dr:  use "wd deploy --check" instead of "check='yes' wd deploy" since gem version 0.5.2

This isn't an open issue so much as a notice about a behavior change that may be surprising to some users.

Basically if you've been doing automated deploys with:

check='yes' wd deploy --to=<project>:<target> ...

As of this commit (which means, as of gem version 0.5.2, released back on September 6, 2010):
  https://github.com/flogic/whiskey_disk/commit/158845c7c18e5aeefb36aa2577360ec08f5bb7b4

You will want to instead do:

wd deploy --check --to=<project>:<target> ...

Otherwise staleness checking will not run.

Note that all the internal wd machinery is a set of 4 rake tasks which use environment variables as controls, and then which call the WhiskeyDisk ruby libraries.  The 'wd' command-line binary is just an args processor which sets environment variables and calls the wd rake tasks.

Setting 'check="yes"' on the command-line is sufficient to turn on staleness checking when running the whiskey_disk rake tasks.  Setting that variable and running the 'wd' command (with or without --check specified) will not, however, pass along the environment setting for "check" to the rake tasks, as the wd binary sets 'check' in the environment based on the presence of absence of the --check flag -- so whatever value is specified in the enclosing environment will always be overwritten.

I'm inclined to think this is the desired behavior, it's just a bit surprising for folks who upgraded through the 0.5.2 release.  Hence the "issue".
